clients/admin-ui/src/features/dataset/DatasetCollectionView.tsx

```diff
@@ -67,8 +67,9 @@ const DatasetCollectionView = ({ fidesKey }: Props) => {
   const activeCollection = useSelector(selectActiveCollection);
   const activeEditor = useSelector(selectActiveEditor);
 
-  const [columns, setColumns] =
-    useState<ColumnMetadata<DatasetField>[]>(ALL_COLUMNS);
+  const [columns, setColumns] = useState<ColumnMetadata<DatasetField>[]>(
+    ALL_COLUMNS.filter((column) => column.name !== "Identifiability")
```

Comment:
perhaps we can just remove `Identifiability` from `ALL_COLUMNS`? I don't think anything else is using it, and we wouldn't want it in the column dropdown either

Reply:
Yep! That's a much easier solution. I wasn't sure if it was helpful to keep around just in case but didn't catch the filter so seems to be cleaner that way 👍🏽

Reply:
Fixed in [0822761](https://github.com/ethyca/fides/pull/2226/commits/082276118aef34f1954aa644505531027d63bcec)!